burnman/solidsolution.py

```diff
@@ -27,7 +27,7 @@ class SolidSolution(Mineral):
     and P derivatives in J/K/mol and m^3/mol.
     """
 
-    def __init__(self, endmembers, solution_model=SolutionModel()):
+    def __init__(self, endmembers, solution_model=SolutionModel(), molar_fraction=None):
```

Comment:
Not a bug (I thing I've been consistent in each file). It's just a horrible legacy from our mega commit in December. We deleted the commit where I fixed this. I can change them all to molar_fractions...